braindecode/models/functions.py

```diff
@@ -372,50 +425,51 @@ def _apply_sinc_resample_kernel(
     width: int,
 ):
     """
-       Apply the sinc resampling kernel.
+     Apply the sinc resampling kernel.
 
-    Notes
-       -----
-       Code copied and modified from TorchAudio.
-
-       All rights reserved.
-       Redistribution and use in source and binary forms, with or without
-       modification, are permitted provided that the following conditions are
-       met:
-           * Redistributions of source code must retain the above copyright
-           notice, this list of conditions and the following disclaimer.
-           * Redistributions in binary form must reproduce the above
-           copyright notice, this list of conditions and the following
-           disclaimer in the documentation and/or other materials provided
-           with the distribution.
-           * Neither the name of the Pytorch Developers nor the names of any
-           contributors may be used to endorse or promote products derived
-           from this software without specific prior written permission.
-       THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
-       "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
-       LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
-       A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
-       OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
-       SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
-       LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
-       DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
-       THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-       (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-       OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
-
-       Parameters
-       ----------
-       waveform: Tensor
-       orig_freq: int
-       new_freq: int
-       gcd: int
-       kernel: Tensor
-       width: int
-
-       Returns
-       -------
-       Tensor
+     Notes
```

Comment:
The formatting is not consistent with the previous function. (Notes should go below the parameters?)

Also, maybe just put the licences once in a separate file and link to it in the notes?